src/pybind/mgr/dashboard/frontend/src/styles.scss

```diff
@@ -33,54 +23,15 @@ $popover-max-width: 350px;
 @import './src/styles/ceph-custom/forms';
 @import './src/styles/ceph-custom/grid';
 @import './src/styles/ceph-custom/icons';
+@import './src/styles/ceph-custom/navs';
+@import './src/styles/ceph-custom/toast';
 
 /* If javascript is disabled. */
 .noscript {
   padding-top: 5em;
 }
 
-.noscript p {
-  color: $color-noscript-text;
-}
-
-/* Card */
-.card-footer button.btn:not(:first-child) {
-  margin-left: 5px;
-}
```

Comment:
You are right, we added ml-x classes to the buttons directly. Somehow they are gone, maybe because of rebasing/merging issue?! However, we will add this for the time being, but to another file.

Reply:
Added via review commit